### PR TITLE
feat(ratio); add ratio for portrait videos #775

### LINKED ISSUE #907: Portrait videos in Boosted 5

### PR DESCRIPTION
**Pull request by MewenLeHo**
Closes #907 

**Preview**: https://deploy-preview-906--boosted.netlify.app/docs/5.1/helpers/ratio/

### CODE REVIEW

File: scss/_variables.scss [hunk 1]
```diff
@@ -642,7 +642,8 @@ $aspect-ratios: (
   "1x1": 100%,
   "4x3": calc(3 / 4 * 100%),
   "16x9": calc(9 / 16 * 100%),
-  "21x9": calc(9 / 21 * 100%)
+  "21x9": calc(9 / 21 * 100%),
+  "9x16": calc(16 / 9 * 100%),
```
Comment (julien-deramond) [inline scss/_variables.scss:646]: * Extra `,` to remove
* A "Boosted mod" comment is needed here

File: site/assets/scss/_component-examples.scss [hunk 2]
```diff
@@ -185,7 +185,7 @@
 .bd-example-ratios {
   .ratio {
     display: inline-block;
-    width: 10rem;
+    width: 9rem;
```
Comment (julien-deramond) [inline site/assets/scss/_component-examples.scss:188]: Missing "Boosted mod" comment

**Review by julien-deramond — CHANGES_REQUESTED** [review]
Some minor comments on aesthetics to fix. Functionally, sounds good to me. @Lausselloic do you want to look at it?

File: scss/_variables.scss (hunk 1 above)
Comment (MewenLeHo) [reply scss/_variables.scss]: Done.

File: site/assets/scss/_component-examples.scss (hunk 2 above)
Comment (MewenLeHo) [reply site/assets/scss/_component-examples.scss]: Done.

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**